data/plugins/treeview.lua

```diff
@@ -819,6 +819,22 @@ command.add(function()
   end
 })
 
+local projectsearch = core.try(require, "plugins.projectsearch")
```

Comment:
IIRC `core.try` creates an error log entry on fail, so it might be better to use `pcall` directly here.
We should probably add an optional `require` function.

Reply:
You are right a third time, I'll put it as pcall for now.